articles/quickstart/native/android/06-calling-apis.md

```diff
@@ -86,6 +119,6 @@ client.newCall(request).enqueue(new Callback() {
 });
 ```
 
-Check if the request was made and if the response that came back was what you expected. 
+Check if the request was made and if the response that came back was what you expected.
 
-You need to configure your server side to protect your API endpoints with the key for your Auth0 client. In this example, you can use the user's access token issued by Auth0 to call Auth0 APIs.
\ No newline at end of file
+You need to configure your backend-app to protect your API endpoints with the key for your Auth0 client, API identifier and API scopes. In this example, you can use the user's access token issued by Auth0 to call your own APIs.
```

Comment:
Pick either `backend-app` or `backend application` and use consistently throughout.  I prefer `backend application`